src/server/models/match.model.js

```diff
@@ -2,22 +2,24 @@ import mongoose from 'mongoose';
 const Schema = mongoose.Schema;
 
 let matchSchema = new Schema({
-  mentorId: Schema.Types.ObjectId,
-  menteeId: Schema.Types.ObjectId,
-  requestDate: {
+  mentor_id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  mentee_id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  request_date: {
     type: Date,
     default: Date.now(),
   },
-  isAccepted: {
-    type: Boolean,
-    default: false,
-  },
-  acceptDate: Date,
-  isCancelled: {
-    type: Boolean,
-    default: false,
+  //status of match {pending:0, accepted:1, rejected:2}
+  status: {
+    type: Number,
+    default: 2,
   },
-  cancelDate: Date,
+  response_date: Date,
```

Comment:
responseDate? 
I don't know what this date means. Is this the date on which the mentee sent a request? Or was this the date that the mentor accepted/rejected the request? The var name should be obvious which it is, and please provide a comment next to this line. It's not clear why we're storing only one of the 2 dates.